core/modules/startup/story.js

```diff
@@ -149,6 +149,7 @@ function openStartupTiddlers(options) {
 	}
 	// Save the story list
 	$tw.wiki.addTiddler({title: DEFAULT_STORY_TITLE, text: "", list: storyList},$tw.wiki.getModificationFields());
+	$tw.wiki.setText(DEFAULT_HISTORY_TITLE,"focus-tiddler",undefined,storyList[0]);
```

Comment:
Hardcoding `DEFAULT_HISTORY_TITLE` imo should be avoided. ... We need to use the `tv-history-list` variable for all of those elements.

Reply:
Same thing with: `DEFAULT_STORY_TITLE`

Reply:
I should be possible to handle several independent stories at the same time

Reply:
Hi @pmario, this is in /startup/story.js , there is no other storylist available

Reply:
That's not true. The story and history tiddlers are defined as parameters of the navigator widget.